validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/PointsetSequences.java

```diff
@@ -105,9 +107,9 @@ private boolean validPointEntry(Entry<String, PointPointsetEvents> point) {
     return point.getValue().present_value != null;
   }
 
-  @Test(timeout = TWO_MINUTES_MS)
+  @Test(timeout = SIX_MINUTES_MS)
```

Comment:
erm? SIX minutes?

Reply:
Following earlier suggestions that this test should support a device which publishes telemetry every five minutes, so six minutes allows the 5 minutes plus setup/teardown.

I just remembered as I was changing these bits, can leave it out of this PR